ansible/roles/passwords/tasks/main.yml

```diff
@@ -7,6 +7,14 @@
   delegate_to: localhost
   run_once: true
 
+- name:  Template k3s token to terraform
+  template:
+    src: k3s-token.auto.tfvars.json
+    dest: "{{ lookup('env', 'APPLIANCES_ENVIRONMENT_ROOT') }}/terraform/k3s-token.auto.tfvars.json"
+  delegate_to: localhost
+  run_once: true
+
+
 # - name: Ensure munge key directory exists
```

Comment:
Given we're making changes here could you just delete this commented out munge-related code pls?